cogment_verse/experiment_tracker/mlflow_experiment_tracker.py

```diff
@@ -37,54 +36,45 @@
 MAX_METRICS_BATCH_SIZE = 1000  # MLFlow only accepts at most 1000 metrics per batch
 
 
-def make_dict(ignore_non_numbers, *args, **kwargs):
-    res = dict(kwargs)
-    for arg in args:
-        if isinstance(arg, Message):
-            arg = MessageToDict(arg, preserving_proto_field_name=True, including_default_value_fields=False)
-        if OmegaConf.is_config(arg):
-            arg = OmegaConf.to_container(arg, resolve=True)
-        if isinstance(arg, dict):
-            for key, value in arg.items():
-                if ignore_non_numbers and not isinstance(value, numbers.Number):
-                    break
-                if key in res:
-                    raise RuntimeError(
-                        f"Trying to set duplicate key [{key}] to [{value}], while already set to [{res[key]}]"
-                    )
-                res[key] = value
-        else:
-            raise RuntimeError(f"Unsupported argument of type [{type(arg)}]")
-    return res
-
-
 class MlflowExperimentTracker:
-    def __init__(self, experiment_id, run_id, mlflow_tracking_uri, flush_frequency=5):
+    def __init__(self, exp_tracker_cfg, experiment_id, run_id):
         self._experiment_id = experiment_id
         self._run_id = run_id
-        self._mlflow_tracking_uri = mlflow_tracking_uri
+        self._mlflow_tracking_uri = exp_tracker_cfg.mlflow_tracking_uri
         self._mlflow_exp_id = None
         self._mlflow_run_id = None
         self._metrics_buffer = []
-        self._flush_metrics_worker_frequency = flush_frequency
+        self._flush_metrics_worker_frequency = exp_tracker_cfg.flush_frequency
         self._flush_metrics_worker = None
 
+        os.environ["MLFLOW_HTTP_REQUEST_BACKOFF_FACTOR"] = str(exp_tracker_cfg.request_backoff_factor)
+        os.environ["MLFLOW_HTTP_REQUEST_MAX_RETRIES"] = str(exp_tracker_cfg.request_max_retries)
+        os.environ["MLFLOW_HTTP_REQUEST_TIMEOUT"] = str(exp_tracker_cfg.request_timeout)
+
     def __del__(self):
         self._stop_flush_metrics_worker()
 
     def _get_mlflow_client(self):
+
         client = MlflowClient(tracking_uri=self._mlflow_tracking_uri)
+
         if not self._mlflow_exp_id:
             mlflow_experiment_name = f"/{self._experiment_id}"
-            experiment = client.get_experiment_by_name(mlflow_experiment_name)
+            try:
```

Comment:
This allows to quickly catch when a user has configured an experiment with the mlflow experiment tracker, without starting the mlflow server.